website/views/options/social/chat-message.jade

```diff
@@ -31,6 +31,9 @@ mixin chatMessages(inbox)
         | &nbsp; &nbsp;
         a(ng-click="flagChatMessage(group._id, message)", ng-if=':: user.contributor.admin || (!message.sent && user.flags.communityGuidelinesAccepted && message.uuid != user.id && message.uuid != "system")')
           span.glyphicon.glyphicon-flag(tooltip="{{message.flags[user._id] ? env.t('abuseAlreadyReported') : env.t('abuseFlag')}}" ng-class='message.flags[user._id] ? "text-danger" : ""')
+        | &nbsp; &nbsp;
+        a(ng-click="copyToDo(message)")
```

Comment:
This appears in the system messages as well. `ng-if='message.uuid != "system"'` should do it, I believe. 

![screen shot 2015-04-25 at 1 52 09 pm](https://cloud.githubusercontent.com/assets/2916945/7334189/4d381d5c-eb52-11e4-8e65-58dea8a85b9c.png)

I can see reasons for wanting to zap your own messages, but not the system ones.